src/sacn/source_state.c

```diff
@@ -208,8 +209,8 @@ int process_sources(process_sources_behavior_t behavior)
 {
   int num_sources_tracked = 0;
 
-  // Iterate the sources backwards to allow for removals
-  for (size_t i = get_num_sources() - 1; i >= 0; --i)
+  // Iterate the sources backwards to allow for removals (i must not be unsigned)
+  for (int i = get_num_sources() - 1; i >= 0; --i)
```

Comment:
I'll have to take another pass and find more ways to switch from unsigned to signed. I was also trying to avoid conversion from signed to unsigned, but in the case of these loops I can make sure the value doesn't go negative, so the conversions should work if/when I need to make them.

Reply:
I'll make these changes in the next PR.